numismatic/feeds/base.py

```diff
@@ -6,8 +6,7 @@
 logger = logging.getLogger(__name__)
 
 
-class Feed(abc.ABC):
-    "Feed Base class"
+class RestApi(abc.ABC):
```

Comment:
Why have both feed and restapi in the same file. Why not have a `feed.py` and a `restApi.py`?

Reply:
Yeah, I just added it as I needed it. Could be in `base_rest.py`. Similarly for `ws_api.py`. 

However given that this will be the template for people to implement feeds and you want each new feed to be contained in one file, shouldn't this be all in one file too?

The file is very small so I think it's manageable.